internal/packager/init.go

```diff
@@ -172,7 +172,10 @@ func initFromComposeFile(name string, composeFile string) error {
 	if err != nil {
 		return errors.Wrap(err, "failed to marshal parameters")
 	}
-	err = ioutil.WriteFile(filepath.Join(dirName, internal.ComposeFileName), composeRaw, 0644)
+	// replace enclosed default values for parameters in the compose file before saving
+	composeStr := removeDefaultValuesFromCompose(string(composeRaw))
```

Comment:
I think we can avoid `bytes`<->`string` translation here, as everything in `removeDefaultValuesFromCompose` should be available for `bytes` (`regexp` package has every functions in pair, `string` or `bytes`, same for `strings.SplitN` which should be `bytes.SplitN`).